lib/pages/home_page.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+
+import 'carteiraApp.dart';
+import 'movimentacoes_page.dart';
+import 'page_screen.dart';
+
+class HomePage extends StatefulWidget {
+  HomePage ({Key? key}) : super(key: key);
+
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+
+  int _indiceAtual = 0;
+  final List<Widget> _telas = [
+
+      NewPageScreen("Pagina Home"),
+      TelaInicialWidget(),
+      MovimentacoesPage()
+
+  ];
+
+
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+
+        body: _telas[_indiceAtual],
```

Comment:
No card é solicitado para utilizar o router nativo e nesse caso você está trocando o body da página, você poderia usar uma função com um switch case para definir para qual rota a tela será direcionada, de acordo com o index do botão da bottom navigation bar que for pressionado. Um exemplo: 
![image](https://user-images.githubusercontent.com/113136109/195603746-066af6a3-c86b-4249-8705-37b7886eebfc.png)

Reply:
nao necessariamente precisa assim kkk desse jeito so é complicado algo simples @guilhermemaiawarren